horace_core/sqw/@Experiment/private/get_mod_pulse_.m

```diff
@@ -16,7 +16,7 @@
 present = true;
 
 % unique_objects cell used to speed access. Should not be used to replace.
-uni_inst = inst.expose_unique_objects();
+uni_inst = inst.expose_unique_objects;
```

Comment:
better as a function, does not look like a property

Reply:
by the way, `expose_unique_objects() looks logicallyu wrong.